actions/buildpack/update/Dockerfile

```diff
@@ -1,20 +0,0 @@
-FROM alpine
-
-RUN apk add \
-    bash \
-    curl \
-  && rm -rf /var/cache/apk/*
-
-RUN mkdir /lib64 && ln -s /lib/libc.musl-x86_64.so.1 /lib64/ld-linux-x86-64.so.2
```

Comment:
is none of this set up needed now?

Reply:
Update - this isn't needed since the action doesn't run in a docker container now.